fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1196,6 +1196,75 @@ public Partition dropPartition(long dbId, String partitionName, boolean isForceD
         return dropPartition(dbId, partitionName, isForceDrop, !isForceDrop);
     }
 
+    public Partition dropPartitionForTruncate(long dbId, boolean isForceDrop,
```

Comment:
OlapTable has a function  dropPartition,  its code seem much like this, can refactor code to reuse them ?

Reply:
thank you. i will refactor it